py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -317,6 +318,10 @@ def compile_module(
     # Assume converters support dynamic shapes and disable validation
     CONVERTERS.set_dynamic_shape_support(settings.assume_dynamic_shape_support)
 
+    # Set non-refitable ops as disallowed targets.
+    if settings.make_refitable:
+        settings.torch_executed_ops.update(REFIT_SENSITIVE_OPS)
```

Comment:
I think its better to do this through validators than maintaining another list

Reply:
1) Currently, the capability validator doesn't have access to user settings. So, I think making the compilation settings in _compiler.py (compile call) a global variable and passing it here would work. https://github.com/pytorch/TensorRT/blob/main/py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py#L439 

2) The other alternative if we don't want to modify user settings is 
```py
CONVERTERS.set_disallowed_targets(settings. REFIT_SENSITIVE_OPS)
```
What would be preferred ?

Reply:
Dont see a drawback to let validators see user settings, could be relevant later like if a certain converter only works for one data type

Reply:
So it would involve updating the converter registry to take the settings as an arg in addition to the node, and some adjustment in the decorator. But overall I think its the best design since we shouldnt be spreading this info in a bunch of global lists, a converter should be able to tell you what it can and cant do on its own 

Dont think we need to make settings global we could make a reference a member of the registry like we do the dynamic shape setting https://github.com/pytorch/TensorRT/blob/4d2a04a7ccd4917f3f15666840d5175f1b9ed9c6/py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py#L300 and then it gets injected into the call to the validator

Reply:
Just an additional arg here: https://github.com/pytorch/TensorRT/blob/4d2a04a7ccd4917f3f15666840d5175f1b9ed9c6/py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py#L439

Reply:
Got it. modified it now.